packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -31,23 +35,40 @@ let {
   ariaInvalid?: boolean | 'grammar' | 'spelling';
   ariaLabel?: string;
   children?: Snippet;
+  multiple?: boolean;
+  label?: string;
 } = $props();
```

Comment:
We need to improve the typing of the onChange method.

https://github.com/podman-desktop/podman-desktop/blob/b087da26a4123323860e5031e198cee8d63be0ed/packages/ui/src/lib/dropdown/Dropdown.svelte#L32

The typing above is not friendly at all, and should be addressed before making it more complex :/